src/test/java/de/tum/in/www1/artemis/util/RequestUtilService.java

```diff
@@ -122,9 +123,35 @@ private void postWithoutLocation(String path, Function<MockHttpServletRequestBui
         restoreSecurityContext();
     }
 
-    public void postWithoutResponseBody(String path, HttpStatus expectedStatus, MultiValueMap<String, String> params) throws Exception {
-        mvc.perform(MockMvcRequestBuilders.post(new URI(path)).params(params)).andExpect(status().is(expectedStatus.value())).andReturn();
+    public MockHttpServletResponse postWithoutResponseBody(String path, HttpStatus expectedStatus, MultiValueMap<String, String> params) throws Exception {
+        MvcResult res = mvc.perform(MockMvcRequestBuilders.post(new URI(path)).params(params)).andExpect(status().is(expectedStatus.value())).andReturn();
         restoreSecurityContext();
+        return res.getResponse();
+    }
+
+    public void postWithoutResponseBody(String path, Object body, HttpStatus expectedStatus) throws Exception {
+        postWithoutResponseBody(path, body, expectedStatus, null, null);
+    }
+
+    public MockHttpServletResponse postWithoutResponseBody(String path, Object body, HttpStatus expectedStatus, @Nullable HttpHeaders httpHeaders) throws Exception {
+        return postWithoutResponseBody(path, body, expectedStatus, httpHeaders, null);
+    }
+
+    public MockHttpServletResponse postWithoutResponseBody(String path, Object body, HttpStatus expectedStatus, @Nullable HttpHeaders httpHeaders,
```

Comment:
Codacy missing comment

Reply:
I can add it but we don't have comments in any methods in this UtilService